api/lib/QueryHandler.js

```diff
@@ -74,9 +74,22 @@ let mergeAddresses = function(addresses) {
   });
 };
 
+const sortMergedTenancies = merged => {
+  sorted_tenancies = merged.tenancies.sort(compareDateStrings);
+  merged.tenancies = { current: [], previous: [] };
```

Comment:
Re-assigning values to arguments is generally a bad idea. Lets return a value from our function instead.